dvc/command/data_sync.py

```diff
@@ -179,6 +179,12 @@ def add_parser(subparsers, _parent_parser):
         default=False,
         help="Fetch run history for all stages.",
     )
+    pull_parser.add_argument(
+        "--glob",
+        action="store_true",
+        default=False,
+        help="Pull cache for targets matching shell-style wildcards.",
```

Comment:
Why not
```suggestion
        help="Allows import paths containing shell-style wildcards.",
```
like in https://github.com/iterative/dvc/pull/5010/files and https://github.com/iterative/dvc/pull/5033/files? For consistency 🙂

Reply:
Is it still ok to say "import" when talking about pull? If so, sure, why not keep the same message.

Reply:
No, sorry. I meant "targets" (instead of "import paths")